agent/lib/kontena/workers/fluentd_worker.rb

```diff
@@ -44,13 +44,10 @@ def on_node_info(topic, info)
     end
 
     def on_log_event(topic, log)
-      # TODO Get more tags
-      # maybe stack.service.instance
-      #puts "forwarding: #{@forwarding}, fluentd: #{@fluentd}"
```

Comment:
BTW: These agent log-on-log things are amusing... you get an infinite log amplification DoS :)

We should figure out some fix for that, there's some parts of the agent that you just can't do any debug logging in ^^